client/scripts/helpers/snapshot_utils/networks.json

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
why is this file necessary? can we just fetch it to the client if necessary?

Reply:
This file is used to get scan explorer url. They did not export this file. So can not fetch it from the client.